main.cpp

```diff
@@ -1,15 +1,18 @@
 #include "main.h"
 #include "cwmods/cwmods.h"
+#include "cwmods/common/LongVector3.h"

```

Comment:
I don't think it should be necessary to include LongVector3 since it's included by cwmods.h?